src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -201,6 +515,16 @@ public void unsupportedExternalBinaryContentConstraint(final String uri) {
                         "this restriction in a resource indicated by a " +
                         "rel=\"http://www.w3.org/ns/ldp#constrainedBy\" link in the Link response header.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        if (externalContentSupported(uri)) {
+            return;
+        }
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUri, "proxy")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.